Error while installing npm dependecies

## Checklist
- [x] I am on the latest version

- [ ] I am on the latest version of dependencies

- [x] I am on the latest version of Discord

- [x] I have followed the instructions in the [Wiki](https://github.com/justdotJS/SimplePresence/wiki)

## Type
Please select what type of issue this is. Please do not use Issues for Support, [ask on the Discord](https://discord.gg/MpnbrX7)!

- [x] Bug Report
- [ ] Feature Request


## Description
When running ´npm install´ on the latest version on macOS Mojave
npm WARN deprecated snekfetch@3.6.4: use node-fetch instead
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/justJS/prism-media.git
npm ERR! 
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/MYNAME/.npm/_logs/2019-02-18T20_27_46_970Z-debug.log


## Specifications
Version is the version of EasyRPC, Platform is Windows/macOS/Linux, and Platform Version is the version of the platform (e.g. 10.13.1 for macOS, or 10 for Windows.).
  - [?] Version: 
  - [macOS] Platform: 
  - [10.14.1] Platform Version:

> @GalacticLion7 did you try removing prism-media from package.json?

Now I'm getting this error after removing the `prism-media` line in package.json:

```
npm ERR! file C:\Users\snipped\SimplePresence\package.json
npm ERR! code EJSONPARSE
npm ERR! JSON.parse Failed to parse json
npm ERR! JSON.parse Unexpected token } in JSON at position 1320 while parsing near '...etch": "^3.5.8",
npm ERR! JSON.parse   },
npm ERR! JSON.parse   "browser": {
npm ERR! JSON.parse ...'
npm ERR! JSON.parse Failed to parse package.json data.
npm ERR! JSON.parse package.json must be actual JSON, not just JavaScript.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\snipped\AppData\Roaming\npm-cache\_logs\2019-03-09T11_51_59_857Z-debug.log
```

I'm using Notepad++, and yes, it's JSON, not JavaScript.

Here's the log:
```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install' ]
2 info using npm@6.4.1
3 info using node@v10.15.3
4 verbose npm-session 61bfb58e9508fdba
5 silly install runPreinstallTopLevelLifecycles
6 silly install loadCurrentTree
7 silly install readLocalPackageData
8 timing stage:rollbackFailedOptional Completed in 2ms
9 timing stage:runTopLevelLifecycles Completed in 417ms
10 verbose stack Error: Failed to parse json
10 verbose stack Unexpected token } in JSON at position 1320 while parsing near '...etch": "^3.5.8",
10 verbose stack   },
10 verbose stack   "browser": {
10 verbose stack ...'
10 verbose stack     at parseError (C:\Program Files\nodejs\node_modules\npm\node_modules\read-package-json\read-json.js:452:11)
10 verbose stack     at parseJson (C:\Program Files\nodejs\node_modules\npm\node_modules\read-package-json\read-json.js:104:26)
10 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\read-package-json\read-json.js:51:5
10 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\graceful-fs\graceful-fs.js:78:16
10 verbose stack     at FSReqWrap.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:53:3)
11 verbose cwd C:\Users\snipped\SimplePresence
12 verbose Windows_NT 10.0.17763
13 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
14 verbose node v10.15.3
15 verbose npm  v6.4.1
16 error file C:\Users\snipped\SimplePresence\package.json
17 error code EJSONPARSE
18 error JSON.parse Failed to parse json
18 error JSON.parse Unexpected token } in JSON at position 1320 while parsing near '...etch": "^3.5.8",
18 error JSON.parse   },
18 error JSON.parse   "browser": {
18 error JSON.parse ...'
19 error JSON.parse Failed to parse package.json data.
19 error JSON.parse package.json must be actual JSON, not just JavaScript.
20 verbose exit [ 1, true ]
```

--- a/package.json
+++ b/package.json
@@ -34,8 +34,7 @@
     "open": "0.0.5",
     "osascript-promise": "^1.0.0",
     "parse-duration": "^0.1.1",
-    "snekfetch": "^3.5.8",
-    "prism-media": "github:justJS/prism-media#patch-1"
+    "snekfetch": "^3.5.8"
   },
   "browser": {
     "net": false,

